File: A_Verify_Password.java
import java.util.*;

public class A_Verify_Password {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int test = sc.nextInt();
        for (int t = 0; t < test; t++) {
            String str = sc.next();
            boolean isValid = true;
            
            List<Character> digits = new ArrayList<>();
            List<Character> letters = new ArrayList<>();
            
            boolean letterStarted = false;
            
            for (int i = 0; i < str.length(); i++) {
                char c = str.charAt(i);
                if (c >= '0' && c <= '9') {
                    if (letterStarted) {
                        isValid = false;
                        break;
                    }
                    digits.add(c);
                } else if (c >= 'a' && c <= 'z') {
                    letters.add(c);
                    letterStarted = true;
                } else {
                    isValid = false; // invalid character found
                    break;
                }
            }
            
            if (isValid && !isSorted(digits)) {
                isValid = false;
            }
            
            if (isValid && !isSorted(letters)) {
                isValid = false;
            }
            
            System.out.println(isValid ? "YES" : "NO");
        }
    }
    
    private static boolean isSorted(List<Character> list) {
        for (int i = 1; i < list.size(); i++) {
            if (list.get(i) < list.get(i - 1)) {
                return false;
            }
        }
        return true;
    }
}

File: A_Verify_Password.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
    int test;
    cin >> test;
    while(test--){
            int len;
            cin >> len;
            string s;
            cin >> s;
            vector< char> v1;
            vector<char > v2;
            int n=s.length();
            bool start=false;
            int idx=0;
            for(int i=0;i<n;i++){
                    if(s[i]>='0' && s[i]<='9'){
                        v1.push_back(s[i]);
                    }
                    if(s[i]>='a' && s[i]<='z'){
                        idx=i;
                        break;
                    }
            }
            bool checking=true;
             for(int i=idx;i<n;i++){
                    if(s[i]>='a' && s[i]<='z'){
                       start=true;
                       v2.push_back(s[i]);
                    }
                    if(s[i]>='0' && s[i]<='9' && start==true){
                        checking=false;
                        break;
                    }
            }
            if(checking==false) cout << "NO"<<'\n';
            else {
                if(is_sorted(v1.begin(),v1.end())  && is_sorted(v2.begin(),v2.end())) cout<<"YES"<<'\n';
                else cout<<"NO\n";
            }

    }
       
    return 0;

}